src/background.js

```diff
@@ -0,0 +1,89 @@
+console.log = function() {}; // disable logging
```

Comment:
I know this existed earlier, but it's here because of a file shift, so I'll comment on it again 😁 

This is dangerous, applies to the entire JS VM, and may screw up other libraries around you! Please consider using a custom local logger in the future, there's lots of options and it would be easy to roll your own too.

Using a custom logger could enable the creation of a "debug" checkbox in `browser_action.html` in the future to enable/disable verbose logging.